### ISSUE TITLE
Unable to build with firebase as import

### ISSUE THREAD

**Issue by jmadler** [developit/preact-cli#87]
I'm unable to build a project that has firebase as an import.  It looks like some of the build steps are preprocessing imported modules in a way that's breaking uglify.

Environment:

  * macOS Sierra 
  * firebase 4.1.1
  * node v7.10.0
  * npm 4.6.1
  * yarn 0.24.5
  * preact-cli 1.1.0
  * preact 8.1.0
  * uglify-js 3.0.14

Reproduction steps:
```shell
$ preact create sample
$ cd sample
$ yarn add firebase
$ edit src/index.js # Add: import firebase from 'firebase';
$ yarn run build
  yarn run v0.24.5
  $ preact build
  bundle.js from UglifyJs
  Unexpected token: punc ()) [bundle.js:5468,33]
  ✨  Done in 6.07s.
```
Here's a [copy of the bundle.js](https://gist.github.com/jmadler/b9e947f1a95e2737161183646ffdd9db#file-bundle-js-L5470).  Note the ```()``` on line 5470 around char 33.

CC @addyosmani @jshcrowthe

**Comment by rkostrzewski** [comment]
```function ka(a,b,c){if(!a)throw Error();``` get's transformed to ```function ka(a,b,c){if(!a)return;();``` because of [ReplacePlugin in webpack config](https://github.com/developit/preact-cli/blob/master/src/lib/webpack-config.js#L381).

@developit is there a way to actually limit it to babel-helpers? I don't think it's a good idea to remove all errors - I mean we all like to throw some errors around when things go south 😅

**Comment by lukeed** [comment]
Oh, nice catch! Went through the plugin source, and there's no option for specifying a file or dir-specific replacement. If anything is possible, it'd have to be thru a webpack context loader...maybe? That, or fork the plugin. It's been idle for a while anyway.

**Comment by addyosmani** [comment]
@lukeed is correct in that plugin doesn't appear to have been active for over a year (no replies to the open PR in that timespan either). If we go down the direction of forking it to address this that doesn't sound like the worst idea :) 

Perhaps there's a cleaner way to limit this to babel-helpers though without needing to do that.

**Comment by lukeed** [comment]
This is something I can take on, and better yet, can justify spending time on it with work.

**Comment by mbleigh** [comment]
I see I'm not the only Googler running into this. Hi @addyosmani, hi @jmadler :wink:

**Comment by lukeed** [comment]
I have time set aside for this tomorrow. New plugin out shortly folks.

**Comment by developit** [comment]
Fix released in 1.1.1! :)
